prow/cmd/build/BUILD.bazel

```diff
@@ -16,8 +16,12 @@ load("@io_bazel_rules_docker//go:image.bzl", "go_image")
 
 go_image(
     name = "image",
-    binary = ":build",
-    visibility = ["//prow:__subpackages__"],
```

Comment:
This was previously using the `distroless` base image (the go_image default). I have updated this to alpine just so it is standardised.

Reply:
We should continue using the distroless images whenever possible. This just adds unnecessary complexity.